### PR TITLE
W-12586676-PlatformRN-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues for MuleSoft Composer] for a list of known issues by date.
 
+== February 27, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* If a Compposer connector supports a default value for a field, that value now appears and is selected in the Composer UI; however, the user can choose to override the default value.
```
Comment (bjoson-mulesoft) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:14]: Composer?

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc [hunk 2]
```diff
@@ -6,6 +6,12 @@ New versions are automatically available once they are released; you don't need
 
 See xref:ms_composer_release_notes_ki.adoc[Known Issues for MuleSoft Composer for Salesforce] for a list of known issues by date.
 
+== February 27, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* If a Compposer connector supports a default value for a field, that value now appears and is selected in the Composer UI; however, the user can choose to override the default value.
```
Comment (bjoson-mulesoft) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:13]: Composer?

**Review by bjoson-mulesoft — APPROVED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc (hunk 1 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:14]:
```suggestion
* If a Composer connector supports a default value for a field, that value now appears and is selected in the Composer UI; however, the user can choose to override the default value.
```

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc (hunk 2 above)
Comment (emileeburkeVT) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:13]:
```suggestion
* If a Composer connector supports a default value for a field, that value now appears and is selected in the Composer UI; however, the user can choose to override the default value.
```

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**